homeassistant/components/binary_sensor/threshold.py

```diff
@@ -13,32 +13,40 @@
 from homeassistant.components.binary_sensor import (
     BinarySensorDevice, PLATFORM_SCHEMA, DEVICE_CLASSES_SCHEMA)
 from homeassistant.const import (
-    CONF_NAME, CONF_ENTITY_ID, CONF_TYPE, STATE_UNKNOWN,
-    ATTR_ENTITY_ID, CONF_DEVICE_CLASS)
+    CONF_NAME, CONF_ENTITY_ID, STATE_UNKNOWN, ATTR_ENTITY_ID,
+    CONF_DEVICE_CLASS)
 from homeassistant.core import callback
 from homeassistant.helpers.event import async_track_state_change
 
 _LOGGER = logging.getLogger(__name__)
 
 ATTR_HYSTERESIS = 'hysteresis'
+ATTR_POSITION = "position"
 ATTR_SENSOR_VALUE = 'sensor_value'
-ATTR_THRESHOLD = 'threshold'
+ATTR_LOWER = 'lower'
+ATTR_UPPER = 'upper'
 ATTR_TYPE = 'type'
 
 CONF_HYSTERESIS = 'hysteresis'
 CONF_LOWER = 'lower'
-CONF_THRESHOLD = 'threshold'
 CONF_UPPER = 'upper'
 
 DEFAULT_NAME = 'Threshold'
 DEFAULT_HYSTERESIS = 0.0
 
-SENSOR_TYPES = [CONF_LOWER, CONF_UPPER]
+POSITION_SENSOR_UNKNOWN = "sensor value unknown"
+POSITION_BELOW = "below"
+POSITION_IN_RANGE = "in range"
+POSITION_ABOVE = "above"
```

Comment:
That is a binary sensor (TRUE/FALSE) and not a sensor. Remove that

Reply:
Why is this wrong? Would knowing what side of the range the value is not have a reasonable use case?